listen-for-shutdown.py

```diff
@@ -1,12 +1,52 @@
 #!/usr/bin/env python
 
-
+import signal
+import sys
 import RPi.GPIO as GPIO
 import subprocess
+import time
+
+PWR_BUTTON_GPIO = 3     # 9(GND) 5-GPIO3(+)
+RST_BUTTON_GPIO = 4     # 9(GND) 7-GPIO4(+)
+LED_STATUS_GPIO = 18    # 6(GND) 12-GPIO18(+)
```

Comment:
These two should be configurable. I'm thinking:

```suggestion
RESET_BUTTON_GPIO = int(os.environ.get('RESET_BUTTON_GPIO', 4))    # 9(GND) 7-GPIO4(+)
LED_STATUS_GPIO = int(os.environ.get('LED_STATUS_GPIO', 18))    # 6(GND) 12-GPIO18(+)
```

*Not sure if casting to int is actually required, and we'd need to import `os` above.*

Reply:
I haven't thought through yet, what is the recommended way for the user to set these since this script runs at startup. Let me know if you have any alternative ideas here.